its/src/test/java/com/sonar/it/scanner/msbuild/others/RuntimeConfigTest.java

```diff
@@ -26,7 +26,7 @@
 import java.nio.file.Paths;
 import org.junit.jupiter.api.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
It is better to use our new custom assertion class everywhere to have custom assertions in scope everywhere. We may add more in the future.
https://joel-costigliola.github.io/assertj/assertj-core-custom-assertions.html#single-assertion-entry-point